recipes/pyuvsim/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "pyuvsim" %}
+{% set version = "0.2.2" %}
+{% set sha256 = "e32e0a765e4c60b6418bb8e97a0f85bb3abb3a7a03932d67ce35922130cbbfbf" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - astropy
+    - mpi4py
+    - numpy 1.14.*
+    - pyuvdata
+    - pyyaml
+    - scipy
+    - six
+  run:
+    - python
+    - astropy
+    - mpi4py
+    - {{ pin_compatible('numpy') }}
+    - pyuvdata
+    - pyyaml
+    - scipy
+    - six
+
+test:
+  requires:
+    - nose
```

Looks like the tests also need:
```suggestion
    - nose
    - h5py
```